### PR TITLE
Add rust example

### PR DESCRIPTION
**Pull request by maxalencar**
Screenshot 2023-03-27 at 15 32 10

Screenshot 2023-03-27 at 14 52 36

### CODE REVIEW

File: rust/src/payment_info.rs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+use serde::{Serialize, Deserialize};
+
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
```
Comment (oliverreid) [inline rust/src/payment_info.rs:4]: Huh I wonder if you could set this as a defulat for the application.

**Review by oliverreid — APPROVED** [review]
Man I have no idea about rust. It looks logical, and fairly intuitive. Nice :D Maybe we should write some more services in it :D

File: rust/src/payment_info.rs (hunk 1 above)
Comment (maxalencar) [reply rust/src/payment_info.rs:4]: yeah no idea, I thought the same, did some google, but couldn't find anywhere, even some people asking the same like here: https://www.reddit.com/r/learnrust/comments/ynpxnq/is_there_a_way_to_make_serde_generate_camelcase/

File: rust/docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+version: "3.9"
+services:
+  demo:
+    build: .
+    ports:
+      - "8080:8080"
+    environment:
+        AUTH_KEY: eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJjb25uZWN0LWUtZGV2QGFwcHNwb3QuZ3NlcnZpY2VhY2NvdW50LmNvbSIsImF1ZCI6Imh0dHBzOi8vZS50ZXN0LmNvbm5lY3QucGF5bWVudHNlbnNlLmNsb3VkIiwiZXhwIjoyMzU1NDIyNDYxLCJpYXQiOjE1OTg1NTg0NjEsInN1YiI6ImNvbm5lY3QtZS1kZXZAYXBwc3BvdC5nc2VydmljZWFjY291bnQuY29tIiwiYXBpS2V5IjoiYTg5NDM5NzAtYjcyZS00MzYwLThlNzgtOWEzZTlhM2ZhYTY1IiwiZW1haWwiOiJjb25uZWN0LWUtZGV2QGFwcHNwb3QuZ3NlcnZpY2VhY2NvdW50LmNvbSJ9.eQA-sAYBkNCQTD90nsdmMM0w0WJWIbMFJMQpOYjD74ov_Bu-b-j1h-LFwSZPhIhLBKmKyAg_tGYWHZch4ukHhNzc4CTLarZr-Acmi4QggWdGjkpTfGfdd_vHr0s06ZrGFeVdR4xLHFve4oFSMOIdR7AZpt62vXPKRSRdhj4W8B5p9qjrXss_xM12CjPuM4rDw4hYcDjG3QyU15a5KmmP108BjGuOevYjq6tcqPxVz0mJ6clf_AgKfxHeIgtYu-u9eVCPr6oNIioIviudcrpaS1ZvE-E6nNRyGfrKLwupS1OMKEpgcXXwkNlhf9cUN8PW4x89oBCqvk8J5i6t5C8Wmg
```
[the remark this answers is not in the document]
Comment (maxalencar) [reply rust/docker-compose.yaml:8]: I personally don't like it, that's the reason the go sample doesn't have it, but all other examples are exposing keys so I didn't even bothered to remve this docker compose file..

https://github.com/Paymentsense/connect-e-samples/blob/master/java/.env
https://github.com/Paymentsense/connect-e-samples/blob/master/dotnet/Paymentsense.Connect-e.Api/appsettings.json
https://github.com/Paymentsense/connect-e-samples/blob/master/php/.env.example

I can remove if you want

[the remark this answers is not in the document]
Comment (maxalencar) [reply rust/docker-compose.yaml]: done

**Review by SethWladyka — APPROVED** [review]

**Pull request merged**